src/libraries/System.IO.MemoryMappedFiles/src/Microsoft/Win32/SafeMemoryMappedFileHandle.Unix.cs

```diff
@@ -5,23 +5,16 @@
 using System.Diagnostics;
 using System.IO;
 using System.IO.MemoryMappedFiles;
-using System.Threading;
 
 namespace Microsoft.Win32.SafeHandles
 {
     public sealed partial class SafeMemoryMappedFileHandle : SafeHandleZeroOrMinusOneIsInvalid
     {
-        /// <summary>
-        /// The underlying FileStream.  May be null.  We hold onto the stream rather than just
-        /// onto the underlying handle to ensure that logic associated with disposing the stream
-        /// (e.g. deleting the file for DeleteOnClose) happens at the appropriate time.
```

Comment:
it's not required anymore (the `DeleteOnClose` logic has been moved from `FileStream` to `SafeFileHandle` in #55153)